pkg/webhook/explorer/decode.go

```diff
@@ -28,11 +28,20 @@ func (d *Decoder) Decode(req Request, into runtime.Object) error {
 	if len(req.Object.Raw) == 0 {
 		return fmt.Errorf("there is no context to decode")
 	}
+	return d.DecodeRaw(req.Object, into)
```

Comment:
Question: Here only decodes `req.Object`, how to decode `observedObject`?

Reply:
We can directly use `DecodeRaw`, for example:
```
e.decoder.DecodeRaw(*req.ObservedObject, observerWorkload)
```
Because `req.Object` is required for all Operation, wo decode it default.

Reply:
I add the comment `// If you want to decode the ObservedObject in the ExploreRequest, use DecodeRaw.` with function `Decode`.